doc/mgr/smb.rst

```diff
@@ -952,3 +952,61 @@ at the prompt. Refer to the `smbclient documentation`_ for more details.
 
 .. _smbclient documentation:
    https://www.samba.org/samba/docs/current/man-html/smbclient.1.html
+
+
+Updating QoS Settings for CephFS-Backed Shares
+=================================================
+
+The SMB manager module now supports updating *QoS (Quality of Service)* settings
+for CephFS-backed shares. This allows administrators to apply per-share rate
+limits on SMB I/O operations, specifically limits on the number of operations
+per second (IOPS) and bandwidth in bytes per second, for both read and
+write operations.
+
+QoS Update Command:
+.. code-block:: console
+
+    ceph smb share update cephfs qos <cluster_id> <share_id> \
+      [--read-iops-limit=<int>] \
+      [--write-iops-limit=<int>] \
+      [--read-bw-limit=<int>] \
+      [--write-bw-limit=<int>]
```

Comment:
This needs to be updated to match the other commands documented in this file and moved (up) to where those are.